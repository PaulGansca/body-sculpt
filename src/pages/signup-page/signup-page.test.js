import { shallow } from 'enzyme';
import React from 'react'
import SignupPage from './signup-page';

let wrapper;

beforeEach(() => {
    wrapper = shallow(<SignupPage />);
})

describe('<SignupPage />', () => {
  it("expect to render signup page", () => {
    expect(wrapper).toMatchSnapshot();
  })
});