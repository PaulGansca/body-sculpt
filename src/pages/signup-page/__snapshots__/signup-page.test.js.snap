// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SignupPage /> expect to render signup page 1`] = `ShallowWrapper {}`;
